pkg/etcdcli/etcdcli.go

```diff
@@ -210,3 +210,27 @@ func (g *etcdClientGetter) UnhealthyMembers() ([]*etcdserverpb.Member, error) {
 
 	return unhealthyMembers, nil
 }
+
+func (g *etcdClientGetter) MemberStatus(member *etcdserverpb.Member) string {
+	cli, err := g.getEtcdClient()
+	if err != nil {
+		klog.Errorf("error getting etcd client: %#v", err)
+		return EtcdMemberStatusUnknown
+	}
+	defer cli.Close()
+
+	if len(member.ClientURLs) == 0 && member.Name == "" {
+		return EtcdMemberStatusNotStarted
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
```

Comment:
fix the rest in this PR too.

Reply:
done, cherry-picked the commit from #137